spring-kafka/src/test/java/org/springframework/kafka/requestreply/ReplyingKafkaTemplateTests.java

```diff
@@ -472,6 +492,35 @@ public void testAggregateNormal() throws Exception {
 		}
 	}
 
+	@Test
+	public void testAggregateNormaStringCorrelationl() throws Exception {
```

Comment:
Nit-pick, but typo in the test method:
```suggestion
	public void testAggregateNormalStringCorrelation() throws Exception {
```
😄